homework_3/homework3.py

```diff
@@ -0,0 +1,26 @@
+#1
+i = " I Love Python"
+print(i*42)
+#2
+age_in_month = 12*17
+print("month =", age_in_month) 
+#3
+age_in_years = 17
+#4
+my_age = "Му name is Dmytro and I’m 17 years old"
```

Comment:
Here you should use string concatenation to put `17` not as a raw value but as a value from the variable `age_in_years`. You can user simple concatenation ("some" + "thing" + some_variable), or f-strings